README.md

```diff
@@ -102,16 +102,16 @@ OR
   rails s
 ```
 
-<!-- ### Run tests
+### Run tests
 
-To run tests, run the following command:
+To run tests, run the following commands:
 
 Example command:
 
-```sh
-  gem install rails rspec
-  rails spec spec/models/
-``` -->
+```
+  bundle install
+  rspec spec
+```
```

Comment:
- [ ] [OPTIONAL] Again for running tests we need to run the migrations and setup the database for test, which is done by running `rails db:migrate RAILS_ENV=test` then only `rspec spec` command will work properly